### PR TITLE
Update users.json

### PR DESCRIPTION
**Pull request by fgeorgatos**

### CODE REVIEW

File: users.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 [
   {
-    "username": "cramakri",
-    "firstName": "Chandrasekhar",
-    "lastName": "Ramakrishnan",
-    "email": "cramakri@ethz.ch",
-    "password": "cramakri"
+    "username": "sdrenku65535",
```
Comment (leafty) [inline users.json]: That's not funny enough, should be `C-3PO` and `R2-D2` with proper gravatars

**Comment by rokroskar** [comment]
why change them here? you can make them robot names where you run the test

File: users.json (hunk 1 above)
Comment (fgeorgatos) [reply users.json]: @leafty : (tms) == too much Star Wars...

File: users.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   {
-    "username": "ableuler",
-    "firstName": "Andreas",
-    "lastName": "Bleuler",
-    "email": "andreas.bleuler@sdsc.ethz.ch",
-    "password": "ableuler"
+    "username": "sdrenku65534",
+    "firstName": "Foo2",
+    "lastName": "Bar2",
+    "email": "noreply@epfl.ch",
```
Comment (ableuler) [inline users.json]: Are you sure that keycloak does not enforce unique email addresses per user? I would assume so...

Comment (fgeorgatos) [reply users.json]: @ableuler : not sure at all, closing this case anyway...

**Comment by fgeorgatos** [comment]
dropped the case for this change, attention is ongoing elsewhere...

**Pull request closed without merging**